### PR TITLE
doc: support `os.sshd.sftp`

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
related to https://github.com/harvester/harvester/issues/4480

### CODE REVIEW

**Review by bk201 — APPROVED** [review]
good from tech perspective

File: docs/install/harvester-configuration.md [hunk 1]
```diff
@@ -425,6 +425,20 @@ os:
     mylabel: myvalue
 ```
 
+### `os.sshd.sftp`
+
+#### Definition
+
+`os.sshd` is used to config the sshd. Now, Harvester only supports the `sftp` as the subparameter of `os.sshd`.
```
Comment (bk201) [inline docs/install/harvester-configuration.md:432]: perhaps say set the field to true enable the SFTP subsystem.

Comment (Vicente-Cheng) [reply docs/install/harvester-configuration.md:432]: Add the comment to the example yaml file.
Thanks for the suggestion!

**Review by bk201 — APPROVED** [review]

File: docs/install/harvester-configuration.md (hunk 1 above)
Comment (jillian-maroket) [inline docs/install/harvester-configuration.md:432]:
```suggestion
Subsystem used to configure the OpenSSH Daemon (sshd). Harvester currently only supports `sftp`.
```

File: docs/install/harvester-configuration.md [hunk 2]
```diff
@@ -425,6 +425,20 @@ os:
...
+
+#### Example
+
+```yaml
+os:
+  sshd:
+    sftp: true  #  means enable the SFTP subsystem
```
Comment (jillian-maroket) [inline docs/install/harvester-configuration.md:439]:
```suggestion
    sftp: true  #  The SFTP subsystem is enabled.
```

**Review by jillian-maroket — COMMENTED** [review]
@Vicente-Cheng Please check the slightly expanded definition.

**Comment by Vicente-Cheng** [comment]
> @Vicente-Cheng Please check the slightly expanded definition.

Hi @jillian-maroket 
Thanks for the review.
Update all suggestions and applied to the v1.3 versioned

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**